client-spark/spark2/src/main/java/org/apache/spark/shuffle/DelegationRssShuffleManager.java

```diff
@@ -133,7 +133,10 @@ private boolean tryAccessCluster() {
     RssConf rssConf = RssSparkConfig.toRssConf(sparkConf);
     List<String> excludeProperties =
         rssConf.get(RssClientConf.RSS_CLIENT_REPORT_EXCLUDE_PROPERTIES);
+    List<String> includeProperties =
+        rssConf.get(RssClientConf.RSS_CLIENT_REPORT_INCLUDE_PROPERTIES);
     rssConf.getAll().stream()
+        .filter(entry -> includeProperties.isEmpty() || includeProperties.contains(entry.getKey()))
```

Comment:
Why do we judge the `includeProperties.isEmpty()`?

Reply:
Empty means include all, that means include config is disabled, it reference what hdfs treat include/excludes datanode list

Reply:
Empty should mean including empty.

Reply:
So how to express including all?

Reply:
If we have include list, you can't express including all.

Reply:
Well, if I set the include option withNoDefaults, do you think it can make sense if we do not set this include option stand for include all?

Reply:
It's ok.

Reply:
Done